pkg/util/client/client.go

```diff
@@ -146,12 +147,22 @@ func PatchStatus(ctx context.Context, c client.Client, obj client.Object, update
 // It returns an error if the patch could not be applied.
 type patchFunc func(patch client.Patch) error
 
-func patchCommon(obj client.Object, updateFn UpdateFunc, patchFn patchFunc, options ...PatchOption) error {
+func patchWithRetry(ctx context.Context, c client.Client, obj client.Object, updateFn UpdateFunc, patchFn patchFunc, options ...PatchOption) error {
 	opts := DefaultPatchOptions()
 	for _, opt := range options {
 		opt(opts)
 	}
-	return executePatch(obj, opts, updateFn, patchFn)
+	fetch := false
```

Comment:
As part of the effort I would like to extend this test: https://github.com/kubernetes-sigs/kueue/blob/9e42c79320a853122908200bae1364db838957ad/pkg/scheduler/scheduler_test.go#L9114

Reply:
Done